### PR TITLE
theme: Add Theme poimandres

### PR DESCRIPTION
**Pull request by that-ambuj**
A very underrated theme ported from vscode:

https://github.com/drcmda/poimandres-theme

image
**Poimandres**

image
**Poimandres Storm**

### CODE REVIEW

**Review by the-mikedavis — COMMENTED** [review]
If the only difference between storm and the normal variant is tuning the palette, can the storm one `inherit` the normal one insted? For example see tokyonight: https://github.com/helix-editor/helix/blob/master/runtime/themes/tokyonight.toml and https://github.com/helix-editor/helix/blob/master/runtime/themes/tokyonight_storm.toml

**Comment by that-ambuj** [comment]
Thanks @the-mikedavis!

I didn't really remember to use `inherits` when I was porting the theme(It must've slipped off my mind). this would help theme and it's variant be more consistent!

File: runtime/themes/poimandres.toml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+"constant.numeric" = { fg = "brightMint" }
+"constant.builtin" = { fg = "hotRed" }
+label = { fg = "gray.c0" }
+string = { fg = "brightMint" }
+
+# Markup
+"markup.heading" = { fg = "brightMint" }
+"markup.bold" = { fg = "bluishGrayBrighter", modifiers = ["bold"] }
+"markup.italic" = { fg = "bluishGrayBrighter", modifiers = ["italic"] }
+"markup.strikethrough" = { modifiers = ["italic", "crossed-out"] }
```
Comment (the-mikedavis) [inline runtime/themes/poimandres.toml:38]:
```suggestion
"markup.strikethrough" = { modifiers = ["italic", "crossed_out"] }
```

**Comment by postsolar** [comment]
Looks awesome but could you please tweak something to make the selection more visible when cursorline is in use? It's barely visible unless extremely zoomed in
image

**Comment by that-ambuj** [comment]
Thanks for the comment @postsolar! I mainly use the storm variant and so I didn't bother much to test drive the regular variant 😅.

I will soon make the required changes.

**Comment by that-ambuj** [comment]
Hey @the-mikedavis, I have made some changes. Please, merge this PR if all looks good to you.

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**